src/librustdoc/html/static/js/search.js

```diff
@@ -1053,9 +1053,17 @@ class RoaringBitmap {
     contains(keyvalue) {
         const key = keyvalue >> 16;
         const value = keyvalue & 0xFFFF;
-        for (let i = 0; i < this.keys.length; ++i) {
-            if (this.keys[i] === key) {
-                return this.containers[i].contains(value);
+        let left = 0;
```

Comment:
Please add some code comments to explain the algorithm used and in particular why we don't simply iterate over the items.

Reply:
Okay, I've added some comments.

Reply:
Much appreciated, thanks!